CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [v4.0.1](https://github.com/spinw/spinw/compare/v4.0.0...v4.0.1)
```

Comment:
Should we revert this back to 4.0.0 or is the cat out of the bag!

Reply:
Oh yes... should revert. I only changed it to manually test that the `release.py` script works.